### ISSUE TITLE
Overflow issue

### ISSUE THREAD

**Issue by max107** [github/government.github.com#641]
![screen shot 2017-12-26 at 13 36 07](https://user-images.githubusercontent.com/862272/34354796-db9e6fe2-ea41-11e7-88cb-45f91130d3de.png)
![screen shot 2017-12-26 at 13 36 17](https://user-images.githubusercontent.com/862272/34354799-df629aae-ea41-11e7-8b22-ef6fba4b5d6a.png)

Solution:

remove negative margins:

```scss
.featured-orgs {
  max-height: 348px;
  overflow: hidden;
  text-align: center;
}
```

**Comment by jbjonesjr** [comment]
any thoughts on this (and the related #642) @sophshep ? Wasn't sure if there was a reason for the negative margins.

### Before:
![image](https://user-images.githubusercontent.com/533967/34463883-40133ee0-ee38-11e7-8d61-d50311f00b5d.png)
Logos are overflow/cropped.

### After: 
![image](https://user-images.githubusercontent.com/533967/34463884-59727586-ee38-11e7-917f-57a346260021.png)

Nicely see them all.

I kind of like the `cropped` look, but some are cropped too much (see the 18F logo for example) where you can't actually tell what it is.

**Comment by jbjonesjr** [comment]
fixed by #679
